file-formats/scp1/type/zif_aff_scp1_v1.intf.abap

```diff
@@ -0,0 +1,300 @@
+      "! A view cluster is a group of maintenance dialogs which are collected in one maintenance unit
+      view_cluster                  TYPE ty_object_type VALUE 'C',
+      "! <p class="shorttext">Logical Transport Object</p>
+      "! Different tables can be grouped together as logical transport objects. Always entire object is exported,
+      "! not only explicitly specified table key
+      logical_transport_object      TYPE ty_object_type VALUE 'L',
+      "! <p class="shorttext">Table (with Text Table)</p>
+      "! All tables must have a generated table maintenance dialog
+      table_with_text_table         TYPE ty_object_type VALUE 'S',
+      "! <p class="shorttext">Individual Transaction Object</p>
+      "! Different tables can be grouped together as individual transport objects.
+      "! Only explicitly specified table keys are exported
+      individual_transaction_object TYPE ty_object_type VALUE 'T',
+      "! <p class="shorttext">View</p>
+      "! All maintenance views must have a generated table maintenance dialog
+      view                          TYPE ty_object_type VALUE 'V',
+    END OF co_object_type.
+
+  "! <p class="shorttext">Record is incomplete</p>
+  TYPES ty_incomplete TYPE c LENGTH 1.
+
+  CONSTANTS:
+    "! <p class="shorttext">Record is incomplete</p>
+    BEGIN OF co_incomplete,
+      "! <p class="shorttext">Record is complete</p>
+      "! BC Sets created using Transport or any API, puts all fields of a data record or data records.
+      "! Such data records are flagged as complete
+      complete   TYPE ty_incomplete VALUE ' ',
+      "! <p class="shorttext">Record is incomplete</p>
+      "! Automatic Customizing Recording only puts the key fields of a data record or
+      "! data records, in the BC Set. Such BC Sets must be post-processed.
+      "! Such data records are flagged as incomplete
+      incomplete TYPE ty_incomplete VALUE 'U',
+    END OF co_incomplete.
+
+  "! <p class="shorttext">Data Record Operation at Activation</p>
+  TYPES ty_operation_at_activation TYPE c LENGTH 1.
+
+  CONSTANTS:
+    "! <p class="shorttext">Data Record Operation at Activation</p>
+    BEGIN OF co_operation_at_activation,
+      "! <p class="shorttext">Insert/Update</p>
+      "! Data records which are to be inserted/updated at activation are flagged with value ''
+      insert_or_update TYPE ty_operation_at_activation VALUE '',
+      "! <p class="shorttext">Delete</p>
+      "! Data records which are to be deleted at activation are flagged with value 'L'
+      delete           TYPE ty_operation_at_activation VALUE 'L',
+    END OF co_operation_at_activation.
+
+  "! <p class="shorttext">Attribute of a field in a data record</p>
+  TYPES ty_additional_info TYPE c LENGTH 3.
+
+  CONSTANTS:
+    "! <p class="shorttext">Attribute of a field in a data record</p>
+    BEGIN OF co_field_attribute,
+      "! <p class="shorttext">Key field that is relevant and has Fixed value</p>
```

Comment:
For titles use title style. 
I am also wondering whether we can have shorter titles here.

Reply:
Followed title style and also renamed to have shorter titles.